packages/scripts/src/create-entrypoint.ts

```diff
@@ -72,15 +91,21 @@ export function createEntrypoint({
     staticBuild,
     publicConfigsRoute,
     config,
+    target,
+    externalFeatures,
 }: ICreateEntrypointsOptions) {
     const configs = getAllValidConfigurations(getConfigLoaders(configurations, mode, configName), envName);
     return `
-import { getTopWindow, FeatureLoadersRegistry, runEngineApp } from '@wixc3/engine-core';
+import * as EngineCore from '@wixc3/engine-core';
+
+const { getTopWindow, FeatureLoadersRegistry, runEngineApp } = EngineCore;
 
 const featureLoaders = new Map(Object.entries({
-    ${createFeatureLoaders(features.values(), envName, childEnvs)}
+    ${createFeatureLoaders(features.values(), envName, childEnvs, target)}
 }));
 
+self.${LOADED_FEATURE_MODULES_NAMESPACE} = {};
```

Comment:
adding on the global context the object that will hold references to all external features